Swift/Collection.swift

```diff
@@ -244,7 +244,7 @@ public final class Collection : CollectionChangeObservable, Indexable {
     /// To remove the listener, call remove() function on the returned listener token
     ///.
     /// If the collection is deleted or the database is closed, a warning message will be logged.
-    public func addChangeListener(listener: @escaping (CollectionChange) -> Void) -> ListenerToken {
+    public func addChangeListener(listener: @escaping (CollectionChange) -> Void) -> ListenerToken? {
```

Comment:
The returned ListenerToken is non-null.

Reply:
It still returns the ListenerToken but there will be no notifications. It's the same behavior when holding the ListenerToken but the database is closed later.